target/stm32f429i-disco/include/target/lcd.h

```diff
@@ -0,0 +1,78 @@
+#pragma once
+/* LCD Size (Width and Height) */
+#define  LCD_SIZE_PIXEL_WIDTH    ((uint16_t)240)
+#define  LCD_SIZE_PIXEL_HEIGHT   ((uint16_t)320)
+
+#define LCD_FRAME_BUFFER       ((uint32_t)0xD0000000)
+#define BUFFER_OFFSET          ((uint32_t)0x50000) 
+
+#define LCD_BACKGROUND_LAYER     0x0000
+
+/** 
+  * @brief  LCD Control pin  
+  */ 
+#define LCD_NCS_PIN             GPIO_Pin_2                  
```

Comment:
Should there be some #includes up at the top of this file to make sure these are already defined?